### PR TITLE
Tiny CLI docs improvements

### PR DESCRIPTION
**Pull request by janpio**
- Commands formatted as code
- Add `build` command to list
- Better description for `prepare` and `compile`
- Fix bug in description of `prepare`
- Fix wrong command used in `cordova compile command` example/spec
- `Vs` => `vs.`
- Better formatting of mutually exclusive options

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,45 +28,51 @@
 
 > The command line tool to build, deploy and manage [Cordova](http://cordova.apache.org)-based applications.
 
-[Apache Cordova](http://cordova.apache.org) allows for building native mobile applications using HTML, CSS and JavaScript. 
+[Apache Cordova](http://cordova.apache.org) allows for building native mobile applications using HTML, CSS and JavaScript.
```
Comment (timbru31) [inline README.md:31]:
```suggestion
[Apache Cordova](https://cordova.apache.org) allows for building native mobile applications using HTML, CSS and JavaScript.
```

File: README.md [hunk 2]
```diff
@@ -28,45 +28,51 @@
 
 > The command line tool to build, deploy and manage [Cordova](http://cordova.apache.org)-based applications.
```
Comment (timbru31) [inline README.md:29]:
```suggestion
> The command line tool to build, deploy and manage [Cordova](https://cordova.apache.org)-based applications.
```

File: README.md [hunk 3]
```diff
@@ -26,47 +26,53 @@
...
 - Cordova allows you to build apps for a number of platforms. Learn more about our [Supported platforms].
 - [Project directory structure] documents the details of the directory structure created by Cordova CLI.
 
-# Contributing
-Cordova is an open source Apache project and contributors are needed to keep this project moving forward. Learn more on 
-[how to contribute on our website][contribute]. 
+## Contributing
+
+Cordova is an open source Apache project and contributors are needed to keep this project moving forward. Learn more on
+[how to contribute on our website][contribute].
```
Comment (timbru31) [inline README.md:73]: This links does not work I assume.

File: README.md [hunk 4]
```diff
@@ -26,47 +26,53 @@
...
 
-# Reporting Issues
+## Reporting Issues
 
 If you find issues with the Cordova CLI, please follow our guidelines for [reporting issues]. Please bear in mind that most of `cordova-cli`'s functionality is implemented in [cordova-lib], so that could be the place to report your issue.
 Platform-specific issues should be reported in the relevant repositories, such as [cordova-android] and [cordova-ios].
```
Comment (timbru31) [inline README.md:78]: Those links do not work either

File: README.md (hunk 3 above)
Comment (erisu) [reply README.md:73]: This link works, it is a reference link.

File: README.md (hunk 4 above)
Comment (erisu) [reply README.md:78]: Same with these

**Pull request merged**